CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.2]
+### Added
+- INSAR GAMMA jobs now expose `include_inc_map` parameter that allows users to include an incidence angle map.
+
+## Fixed
```

Comment:
This is probably `Changed` rather than `Fixed`, since there was never a problem in develop/main.  Could link https://github.com/ASFHyP3/hyp3/issues/341